showcase/src/Tables/Stories.elm

```diff
@@ -3,218 +3,103 @@ module Tables.Stories exposing (stories, update)
-            , isSelected = isSelected
-            , items = items
-            }
-
-        isSelected { name } =
-            selectedSomeone
-                |> Maybe.map (.name >> (==) name)
-                |> Maybe.withDefault False
-
-
-        mobileCover { name } =
-            { title = name, caption = Nothing }
-
-        mobileDetails =
-            cellMobileDetailsEnd
-                |> cellMobileDetailsShow
-                |> cellMobileDetailsHide
-
-    in
-    table headers
-        |> Table.withResponsiveRows responsiveOpt
-        |> Table.withWidth fill
-        |> Table.withCellsWidth cellsWidth
-        |> Table.withCellsDetails mobileDetails
-        |> Table.renderElement cfg
-```"""
-          }
+        ( "Mobile"
+        , \{ tablesStories } -> demoWithIcons mobileCfg tablesStories
+        , { note = "See [docs](https://package.elm-lang.org/packages/PaackEng/paack-ui/latest/UI-Table) for the exact code of this example." }
         )
 
 
-staticTable cfg =
+demoTable renderConfig model =
     let
-        headers =
-            header "HEADER A" <| header "HEADER B" <| header "HEADER C" <| header "HEADER D" <| headersEnd
-
-        rows =
-            [ cell "FIRST-CELL A1" <| cell "CELL B1" <| cell "CELL C1" <| cell "LAST-CELL D1" <| rowEnd
-            , cell "FIRST-CELL A2" <| cell "CELL B2" <| cell "CELL C2" <| cell "LAST-CELL D2" <| rowEnd
-            , rowEnd
-                -- Always in reverse so it's O(1) and not O(N)
-                |> cell "LAST-CELL D3"
-                |> cell "CELL C3"
-                |> cell "CELL B3"
-                |> cell "FIRST-CELL A3"
-            ]
-
-        cellsWidth =
-            cellWidthPortion 2 <| cellWidthPortion 1 <| cellWidthPortion 1 <| cellWidthPortion 2 <| cellWidthEnd
-
-        cell str =
-            Text.body1 str |> cellFromText
+        tableColumns =
+            columnsEmpty
+                |> columnsPush (headerToColumn "Title" |> columnWidthPixels 320)
+                |> columnsPush (headerToColumn "Author" |> columnWidthPixels 240)
+                |> columnsPush (headerToColumn "Year" |> columnWidthPixels 120)
+
+        toTableRow { author, title, year } =
+            rowEmpty
+                |> rowPushText (Text.body1 title)
+                |> rowPushText (Text.body2 author)
+                |> rowPushText (Text.caption year)
+
+        toTableDetails { author, title } =
+            detailsEmpty
+                |> detailsPushHidden
+                |> detailsPush { label = "Author", content = cellFromText <| Text.body2 author }
+                |> detailsPushHidden
+
+        toTableCover { title, year } =
+            { title = title, caption = Just year }
+
+        someFilters =
+            filtersEmpty
+                |> filtersPushSingleText Nothing (filterLocal (\{ title } str -> String.contains str title))
+                |> filtersPushSingleText (Just "Dan") (filterLocal (\{ author } str -> String.contains str author))
```

Comment:
So, why not implement this function inside of the Table? why this local filter would be different?

Also, this function is case sensitive, I think it should be case insensitive